content/swift/concepts/dictionaries/terms/contains/contains.md

```diff
@@ -1,47 +1,50 @@
 ---
-Title: '.contains()'
-Description: 'Returns a Boolean value indicating whether the sequence contains the given element.'
+Title: 'contains'
+Description: 'A Boolean value that indicates whether a dictionary contains a specified key.'
 Subjects:
-  - 'Computer Science'
   - 'Mobile Development'
+  - 'Computer Science'
 Tags:
   - 'Dictionary'
   - 'Properties'
-  - 'Arrays'
 CatalogContent:
   - 'learn-swift'
   - 'paths/build-ios-apps-with-swiftui'
 ---
 
-The **`.contains()`** method returns `true` if an array contains a specified element, and `false` otherwise.
+The **`.contains`** method returns a Boolean value that indicates whether a dictionary contains a specified key. The method takes a single argument, which is a key. If the dictionary contains the specified key, the method returns `true` otherwise, it returns `false`.
```

Comment:
```suggestion
The **.contains()** method returns a Boolean value that indicates whether a dictionary contains a specified key. The method takes a single argument, which is a key. If the dictionary contains the specified key, the method returns `true` otherwise, it returns `false`.
```